content/en/guides/hosting/iam/scim.md

```diff
@@ -788,17 +969,87 @@ POST /scim/Groups
 - **Endpoint**: **`<host-url>/scim/Groups/{id}`**
 - **Method**: PATCH
 - **Description**: Update an existing team's membership list.
-- **Supported Operations**: `add` member, `remove` member
+- **Supported Operations**: `add` member, `remove` member, `replace` members
 
 {{% alert %}}
 The remove operations follow RFC 7644 SCIM protocol specifications. Use the filter syntax `members[value eq "{user_id}"]` to remove a specific user, or `members` to remove all users from the team.
+
+**User Identification**: The `{user_id}` in member operations can be either:
+- A W&B user ID
+- An email address (e.g., "user@example.com")
 {{% /alert %}}
 
 
 {{% alert color="info" %}}
-Replace `{team_id}` with the actual team ID and `{user_id}` with the actual user ID in your requests.
+Replace `{team_id}` with the actual team ID and `{user_id}` with the actual user ID or email address in your requests.
 {{% /alert %}}
 
+### Replace team members
+
+Replaces all members of a team with a new list.
+
+- **Endpoint**: **`<host-url>/scim/Groups/{id}`**
+- **Method**: PUT
+- **Description**: Replace the entire team membership list.
+
+{{< tabpane text=true >}}
+{{% tab header="Request" %}}
+```bash
+PUT /scim/Groups/{team_id}
+```
+
+```json
+{
+    "schemas": ["urn:ietf:params:scim:schemas:core:2.0:Group"],
+    "displayName": "wandb-devs",
+    "members": [
+        {
+            "value": "{user_id_1}"
+        },
+        {
+            "value": "{user_id_2}"
+        }
+    ]
+}
+```
+{{% /tab %}}
+{{% tab header="Response" %}}
+```bash
+(Status 200)
+```
+
+```json
+{
+    "displayName": "wandb-devs",
+    "id": "ghi",
+    "members": [
+        {
+            "Value": "user_id_1",
+            "Ref": "",
```

Comment:
What are these fields for? Must we include them if they are empty? Would it be better to show some actual values?